pylzy/lzy/py_env/py_env_provider.py

```diff
@@ -378,3 +376,51 @@ def _get_module_parents(module: ModuleType) -> Iterable[ModuleType]:
         yield parent
 
         parts.pop()
+
+
+def _check_url_is_local_file(url: str) -> bool:
+    file_scheme = 'file://'
+
+    if not url.startswith(file_scheme):
+        return False
+
+    raw_path = url[len(file_scheme):]
+
+    # idk if there may be relative paths in direct_url.json,
+    # but if it is, next code will work badly with it.
+    if not raw_path.startswith('/'):
```

Comment:
This path in windows can contain '\' and other windows stuff.
We can use os.path here or just skip this check